client/apps/edit/components/admin/index.coffee

```diff
@@ -457,8 +457,8 @@ module.exports = class EditAdmin extends Backbone.View
     clientFormat = moment(date).format('L')
     publishTime = moment(date).format('HH:mm')
     @saveFormat = moment(date).toDate()
-    $('.edit-admin-input-date').val(clientFormat)
-    $('.edit-admin-input-time').val(publishTime)
+    $('#edit-admin-publish-date input').val(clientFormat)
+    $('#edit-admin-schedule-publish-time input').val(publishTime)
 
```

Comment:
This stuff is unrelated - I'm trying to track down a bug where an 'Invalid Date' gets appended to the tags/social share section. Trying to be more specific with the inputs.